### PR TITLE
Yell about TypeScript errors more aggressively

### PR DESCRIPTION
**Pull request by nathanleclaire**
Fixes https://github.com/workbenchapp/solana-workbench/issues/75

### CODE REVIEW

File: src/main/types.d.ts [hunk 1]
```diff
@@ -1,2 +0,0 @@
-declare module 'randomart';
```
Comment (nathanleclaire) [inline src/main/types.d.ts:1]: This was a weird one, originally I had to add these to get TS compiler to stop complaining that types couldn't be found for these modules, but then it started complaining again for some reason despite the existence of this file when I removed the transpile only false settings. 🤷 

It works and we're looking to rm these modules anyway so I'm rolling with the switch to require for now

File: package.json [hunk 2]
```diff
@@ -149,30 +149,29 @@
     ]
   },
   "devDependencies": {
+    "@pmmmwh/react-refresh-webpack-plugin": "0.5.4",
+    "@solana/web3.js": "^1.31.0",
```
Comment (nathanleclaire) [inline package.json:153]: Bleh, these changes are more far-reaching than I intended, sorry. `npm install --save-dev` for a new `@types` file that the CI was breaking due to not having ended up making all these changes.

**Comment by SvenDowideit** [comment]
lets see what `docker build .` does for me :)

**Comment by SvenDowideit** [comment]
ok, succeeds on local new clone - so its something-something out of sync (DEFER fixing that)

**Comment by SvenDowideit** [comment]
ok, so the expected effect is that the `npm exec tsc` errors will now also be shown in `npm start` - no change to `npm run lint`.

Hopefully there's a way to make them also surface in `npm run build` (it doesn't seem to happen for me atm)

LGTM

**Review by SvenDowideit — APPROVED** [review]
LGTM!

**Pull request merged**